### ISSUE TITLE
Cross-reference

### ISSUE THREAD

**Issue by PierreSolomon** [data-intuitive/quarto-d2#5]
Hello,

Is there a way to cross-reference the diagram generated by the extension?

**Comment by rcannood** [comment]
Good point! I need to check but this may be an oversight from my part. You should be able to manually add a label in the caption and cite that, but I should work on supporting it in the same way quarto does.

**Comment by elipousson** [comment]
I think using the FloatRefTarget AST node is the way to do this https://quarto.org/docs/prerelease/1.4/lua_changes.html#the-floatreftarget-ast-node
